### PR TITLE
Initial draft of the coordination stakeholder's decision at intake

### LINKED ISSUE #39: Define coordinator decision points and values and enumerate outcomes for them
- Enumerate outcomes for coordinators 
- Define decision points for coordinators; share with existing decision points as much as plausible
- Define decision values for any new decision points that need to be created
- Assign an outcome to each combination of decision values

### PR DESCRIPTION
**Pull request by j---**
Related to #11 
Closes #39 

Notes a TODO for the publication decision, but that is not addressed here.

### CODE REVIEW

**Comment by ahouseholder** [comment]
(I have a few things, I'll do them in separate comments.)

Report Credibility seems like it might be a good place to include Scope. A report must be both credible and in scope to proceed (for whatever your coordination scope is) If either not credible or not in scope or both gets a "no". The structure of the question and its answers doesn't really change.

Unless you wanted to have decorators on coordinate, in which case you might have a "coordinate (handoff)" for credible-but-out-of-scope.

**Comment by ahouseholder** [comment]
I think "Vendor Engagement" is a better name for "Vendor Cooperation" here, for reasons that will be more clear in an anticipated future pull request.

File: doc/version_1/048_coordination.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+The two decisions that CERT/CC makes as a coordinator that we will discuss in terms of SSVC are the initial triage of vulnerability reports and whether a publication about a vulnerability is warranted.
+The initial coordination decision is a prioritization decision, but it does not have the same values as prioritization by a deployer or supplier.
+The publication decision for us is a binary yes/no.
+These two decisions are not the entirety of vulnerability coordination, but we leave further details of the process for future work.
+
+## Coordination Triage Decisions
+
+We take three priority levels in our decision about whether and how to coordinate a vulnerability based on an incoming report:
+
```
Comment (laurie-tyz) [inline doc/version_1/048_coordination.md:22]: Do we describe 'how to coordinate'?

File: doc/version_1/048_coordination.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+  - Credible
+  - Not credible
+
+An analyst should start with a presumption of credibility and proceed toward disqualification.
+The reason for this is that, as a coordinator, occasionally doing a bit of extra work on a bad report is preferable to rejecting legitimate reports.
+This is essentially stating a preference for false positives over false negatives with respect to credibility determination.
+
+The following subsections provide suggestive guidance on assessing credibility.
+There are not ironclad rules for this assessment, and other coordinators may find other guidance works for them.
```
Comment (laurie-tyz) [inline doc/version_1/048_coordination.md:82]: s/not/no
"There are no ironclad rules"

**Comment by laurie-tyz** [comment]
> (I have a few things, I'll do them in separate comments.)
> 
> Report Credibility seems like it might be a good place to include Scope. A report must be both credible and in scope to proceed (for whatever your coordination scope is) If either not credible or not in scope or both gets a "no". The structure of the question and its answers doesn't really change.
> 
> Unless you wanted to have decorators on coordinate, in which case you might have a "coordinate (handoff)" for credible-but-out-of-scope.
-------------------------------------
The first "no" (wins) -> 
? Credible -> no  <STOP>  The scope doesn't matter.  (Depending on the order of the questions...)

**Review by j--- — COMMENTED** [review]
I think https://github.com/CERTCC/SSVC/pull/94/commits/4f1e2176ae182317b8af6ad5bd25550451e40a7d addresses all the comments here.

**Pull request merged**